readme.md

```diff
@@ -214,6 +215,29 @@ Kirby comes bundled with a default set of icons. Make sure the `.svg` files used
 }
 ```
 
+### Fonts
+
+Kirby comes bundled with the [Roboto][roboto] font. Make sure the `.woff2` files used by Kirby are copied to your output folder by adding the following to `build > options > assets` in `angular.json`:
+
+```json
+{
+  ...
+  "build": {
+    "options": {
+      "assets": [
+        ...,
+        {
+          "glob": "**/*.woff2",
+          "input": "node_modules/@kirbydesign/designsystem/fonts",
+          "output": "./assets/kirby/fonts"
+        },
+        ...
+      ],
+    }
+  }
+}
+```
+
```

Comment:
No longer relevant after importing from @fontsource
```suggestion
```

```

Reply:
should be resolved by 3d0f90a